src/server/api/routes/answers.router.js

```diff
@@ -76,35 +76,68 @@ router.get('/:id', (req, res, next) => {
  *        schema:
  *          type: object
  *          required:
- *            - title
- *            - startDate
- *            - endDate
- *             - classId
+ *            - answer
+ *            - ux_points
+ *            - fullstack_points
+ *            - dataanalyst_points
+ *            - fk_question_id
  *          properties:
- *            title:
- *              type: string
- *            startDate:
- *              type: string
- *              format: date-time
- *            endDate:
- *              type: string
- *              format: date-time
- *            classId:
+ *            answer:
  *              type: string
+ *            ux_points:
+ *              type: integer
+ *            fullstack_points:
+ *              type: integer
+ *            dataanalyst_points:
+ *              type: integer
+ *            fk_question_id:
+ *              type: integer
  *    responses:
  *      201:
- *        description: Answer created
+ *        description: answer created
  *      5XX:
  *        description: Unexpected error.
  */
 router.post('/', (req, res) => {
   answersController
-    .createAnswer(req.body)
+    .createNewAnswer(req.body)
```

Comment:
the word new is redundant, you don't create old or non-new stuff, so createAnswer is enough